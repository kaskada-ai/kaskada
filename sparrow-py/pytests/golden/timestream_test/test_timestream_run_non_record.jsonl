{"_time":"1996-12-19T16:39:57.000","_subsort":0,"_key_hash":12960666915911099378,"_key":"A","result":5.0}
{"_time":"1996-12-19T16:39:58.000","_subsort":1,"_key_hash":2867199309159137213,"_key":"B","result":24.0}
{"_time":"1996-12-19T16:39:59.000","_subsort":2,"_key_hash":12960666915911099378,"_key":"A","result":17.0}
{"_time":"1996-12-19T16:40:00.000","_subsort":3,"_key_hash":12960666915911099378,"_key":"A","result":null}
{"_time":"1996-12-19T16:40:01.000","_subsort":4,"_key_hash":12960666915911099378,"_key":"A","result":12.0}
{"_time":"1996-12-19T16:40:02.000","_subsort":5,"_key_hash":12960666915911099378,"_key":"A","result":null}
