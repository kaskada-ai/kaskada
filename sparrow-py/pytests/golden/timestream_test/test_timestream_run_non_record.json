{"_time":851042397000000000,"_subsort":0,"_key_hash":12960666915911099378,"_key":"A","result":5.0}
{"_time":851042398000000000,"_subsort":1,"_key_hash":2867199309159137213,"_key":"B","result":24.0}
{"_time":851042399000000000,"_subsort":2,"_key_hash":12960666915911099378,"_key":"A","result":17.0}
{"_time":851042400000000000,"_subsort":3,"_key_hash":12960666915911099378,"_key":"A","result":null}
{"_time":851042401000000000,"_subsort":4,"_key_hash":12960666915911099378,"_key":"A","result":12.0}
{"_time":851042402000000000,"_subsort":5,"_key_hash":12960666915911099378,"_key":"A","result":null}
